backend/src/test/getHistoricalBalance.test.ts

```diff
@@ -0,0 +1,111 @@
+import { PostgreSqlContainer, type StartedPostgreSqlContainer } from '@testcontainers/postgresql';
+import type { FastifyInstance } from 'fastify';
+import { afterAll, beforeAll, describe, expect, test } from 'vitest';
+
+import { buildApp } from '@/app';
+import * as schema from '../db/schema';
+
+describe('GET /get_historical_balance route', () => {
+  let container: StartedPostgreSqlContainer;
+  let app: FastifyInstance;
+  const testAddress = '0x064a24243f2aabae8d2148fa878276e6e6e452e3941b417f3c33b1649ea83e11';
+
+  beforeAll(async () => {
+    container = await new PostgreSqlContainer().start();
+    const connectionUri = container.getConnectionUri();
+    app = buildApp({
+      database: {
+        connectionString: connectionUri,
+      },
+      app: {
+        port: 8080,
+      },
+    });
+
+    await app.ready();
+    await app.db.insert(schema.usdcBalance).values([
+      {
+        balanceId: 'YfuanNvdfMfcmmdeklWDJO',
+        address: testAddress,
+        blockTimestamp: new Date('2024-04-10 13:03:05'),
+        balance: '3.8AE83A109D0635A426BB',
```

Comment:
don't refer to the spec for the balance it's actually not hex but regular base 10

Reply:
the numbers should be formatted like that `1.234567`

Reply:
Alright